IM.py

```diff
@@ -0,0 +1,56 @@
+"""The purpose of this file is to illustrate how to process Spotter wave spectra to calculate wind speed using the
+IM (inverse modelling) method in Dorsay et al. 2023."""
+
+from roguewave import load
+from roguewave.wavephysics.windestimate import estimate_u10_from_source_terms
+from roguewave.wavephysics.balance import create_balance
+import matplotlib.pyplot as plt
+import pandas as pd
+
+spotter_id = "SPOT-010340"
+
+### Define method parameters
+im_params = load('s3://sofar-wx-data-dev-os1/proxy_wind/calibration/final_calibration_st4_weighted=True.zip')
+methods_to_plot = {
+    'st4': {
+        'method': 'st4',
+        'params': im_params,
+        'recalc': False
+    }
+}
+
+### Get Spotter data, output format pandas DataFrame
+wave_spectra = load('./data/SPOT-010340_spectra_1d')
+
+### Convert Spotter frequency data to 2D spectra to feed into estimation function
+wave_spectra_2d = wave_spectra.as_frequency_direction_spectrum(36)
+
+### Calculate wind estimates for 1 month of Spotter data along Spotter's track
+balance = create_balance(methods_to_plot['st4']['method'], methods_to_plot['st4']['method'])
```

Comment:
the methods to plot doesn't really add anything- I would just directly substitute the values in here